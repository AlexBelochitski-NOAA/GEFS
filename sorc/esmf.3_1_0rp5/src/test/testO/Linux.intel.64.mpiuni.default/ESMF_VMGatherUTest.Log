20190610 112233.797901 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112233.798150 INFO     PET0  Beginning Test, file ESMF_VMGatherUTest.F90, line          67
20190610 112233.798167 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112233.798208 INFO     PET0  PASS  Gather() Test for Integer, ESMF_VMGatherUTest.F90, line         108
20190610 112233.798218 INFO     PET0  PASS  Verifying array1 data after Gather() Test for Integer, ESMF_VMGatherUTest.F90, line         125
20190610 112233.800561 INFO     PET0  PASS  Verifying array2 data after Gather() Test for Integer, ESMF_VMGatherUTest.F90, line         142
20190610 112233.800570 INFO     PET0  PASS  Gather() Test for ESMF_KIND_R8, ESMF_VMGatherUTest.F90, line         153
20190610 112233.800591 INFO     PET0  PASS  Verifying farray1 data after Gather() Test for ESMF_KIND_R8, ESMF_VMGatherUTest.F90, line         170
20190610 112233.800665 INFO     PET0  PASS  Verifying farray2 data after Gather() Test for ESMF_KIND_R8, ESMF_VMGatherUTest.F90, line         192
20190610 112233.800673 INFO     PET0  PASS  Gather() Test for ESMF_KIND_R4, ESMF_VMGatherUTest.F90, line         203
20190610 112233.800754 INFO     PET0  PASS  Verifying f4array1 data after Gather() Test for ESMF_KIND_R4, ESMF_VMGatherUTest.F90, line         220
20190610 112233.800831 INFO     PET0  PASS  Verifying f4array2 data after Gather() Test for ESMF_KIND_R4, ESMF_VMGatherUTest.F90, line         242
20190610 112233.800840 INFO     PET0  PASS  GatherV() Test for Integer, ESMF_VMGatherUTest.F90, line         284
20190610 112233.800850 INFO     PET0  PASS  Verifying array1 data after GatherV() Test for Integer, ESMF_VMGatherUTest.F90, line         301
20190610 112233.800879 INFO     PET0  PASS  Verifying array2 data after GatherV() Test for Integer, ESMF_VMGatherUTest.F90, line         318
20190610 112233.800890 INFO     PET0  PASS  GatherV() Test for ESMF_KIND_R8, ESMF_VMGatherUTest.F90, line         329
20190610 112233.800899 INFO     PET0  PASS  Verifying farray1 data after GatherV() Test for ESMF_KIND_R8, ESMF_VMGatherUTest.F90, line         346
20190610 112233.800932 INFO     PET0  PASS  Verifying farray2 data after GatherV() Test for ESMF_KIND_R8, ESMF_VMGatherUTest.F90, line         368
20190610 112233.800950 INFO     PET0  PASS  GatherV() Test for ESMF_KIND_R4, ESMF_VMGatherUTest.F90, line         379
20190610 112233.800960 INFO     PET0  PASS  Verifying f4array1 data after GatherV() Test for ESMF_KIND_R4, ESMF_VMGatherUTest.F90, line         396
20190610 112233.801007 INFO     PET0  PASS  Verifying f4array2 data after GatherV() Test for ESMF_KIND_R4, ESMF_VMGatherUTest.F90, line         418
20190610 112233.801024 INFO     PET0  Number of failed tests:           0
20190610 112233.801031 INFO     PET0  Ending Test, file ESMF_VMGatherUTest.F90, line         431
