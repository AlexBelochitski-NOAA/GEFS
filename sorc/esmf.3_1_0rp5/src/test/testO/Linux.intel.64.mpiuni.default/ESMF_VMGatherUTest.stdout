  Beginning Test, file ESMF_VMGatherUTest.F90, line          67
  NUMBER_OF_PROCESSORS           1
  PASS  Gather() Test for Integer, ESMF_VMGatherUTest.F90, line         108
  PASS  Verifying array1 data after Gather() Test for Integer, ESMF_VMGatherUTes
 t.F90, line         125
 Contents after gather (should be 1,2,... at 
 localPet=petCount-1 , 0 elsewhere), :
           0  array1:            1
           0  array1:            2
  PASS  Verifying array2 data after Gather() Test for Integer, ESMF_VMGatherUTes
 t.F90, line         142
  PASS  Gather() Test for ESMF_KIND_R8, ESMF_VMGatherUTest.F90, line         153
  PASS  Verifying farray1 data after Gather() Test for ESMF_KIND_R8, ESMF_VMGath
 erUTest.F90, line         170
 contents after gather (should be 1.,2.,... at 
 localPet=petCount-1 , 0. elsewhere), :
           0  farray1:    1.00000000000000     
           0  farray1:    2.00000000000000     
 contents after gather (should be localPet*2+1, localPet*2+2):
           0  farray2:    1.00000000000000     
           0  farray2:    2.00000000000000     
  PASS  Verifying farray2 data after Gather() Test for ESMF_KIND_R8, ESMF_VMGath
 erUTest.F90, line         192
  PASS  Gather() Test for ESMF_KIND_R4, ESMF_VMGatherUTest.F90, line         203
  PASS  Verifying f4array1 data after Gather() Test for ESMF_KIND_R4, ESMF_VMGat
 herUTest.F90, line         220
 contents after gather (should be 1.,2.,... at 
 localPet=petCount-1 , 0. elsewhere), :
           0  f4array1:    1.000000    
           0  f4array1:    2.000000    
 contents after gather (should be localPet*2+1, localPet*2+2):
           0  f4array2:    1.000000    
           0  f4array2:    2.000000    
  PASS  Verifying f4array2 data after Gather() Test for ESMF_KIND_R4, ESMF_VMGat
 herUTest.F90, line         242
  PASS  GatherV() Test for Integer, ESMF_VMGatherUTest.F90, line         284
  PASS  Verifying array1 data after GatherV() Test for Integer, ESMF_VMGatherUTe
 st.F90, line         301
 Contents after gather (should be 1,2,... at 
 localPet=petCount-1 , 0 elsewhere), :
           0  array1:            1
           0  array1:            2
  PASS  Verifying array2 data after GatherV() Test for Integer, ESMF_VMGatherUTe
 st.F90, line         318
  PASS  GatherV() Test for ESMF_KIND_R8, ESMF_VMGatherUTest.F90, line         32
 9
  PASS  Verifying farray1 data after GatherV() Test for ESMF_KIND_R8, ESMF_VMGat
 herUTest.F90, line         346
 contents after gather (should be 1.,2.,... at 
 localPet=petCount-1 , 0. elsewhere), :
           0  farray1:    1.00000000000000     
           0  farray1:    2.00000000000000     
 contents after gather (should be localPet*2+1, localPet*2+2):
           0  farray2:    1.00000000000000     
           0  farray2:    2.00000000000000     
  PASS  Verifying farray2 data after GatherV() Test for ESMF_KIND_R8, ESMF_VMGat
 herUTest.F90, line         368
  PASS  GatherV() Test for ESMF_KIND_R4, ESMF_VMGatherUTest.F90, line         37
 9
  PASS  Verifying f4array1 data after GatherV() Test for ESMF_KIND_R4, ESMF_VMGa
 therUTest.F90, line         396
 contents after gather (should be 1.,2.,... at 
 localPet=petCount-1 , 0. elsewhere), :
           0  f4array1:    1.000000    
           0  f4array1:    2.000000    
 contents after gather (should be localPet*2+1, localPet*2+2):
           0  f4array2:    1.000000    
           0  f4array2:    2.000000    
  PASS  Verifying f4array2 data after GatherV() Test for ESMF_KIND_R4, ESMF_VMGa
 therUTest.F90, line         418
  Number of failed tests:           0
  Ending Test, file ESMF_VMGatherUTest.F90, line         431
